doc/modules/calibration.rst

```diff
@@ -30,16 +30,20 @@ approximately 80% actually belong to the positive class.
 Calibration curves
 ------------------
 
-Calibration curves (also known as reliability diagrams) compare how well the
-probabilistic predictions of a binary classifier are calibrated. It plots
-the true frequency of the positive label against its predicted probability,
-for binned predictions.
-The x axis represents the average predicted probability in each bin. The
-y axis is the *fraction of positives*, i.e. the proportion of samples whose
-class is the positive class (in each bin). The top calibration curve plot
-is created with :func:`CalibrationDisplay.from_estimators`, which uses
-:func:`calibration_curve` to calculate the per bin average predicted
-probabilities and fraction of positives.
+Calibration curves, also referred to as the *reliability diagrams* (Wilks 1995 [2]_),
+compare how well the probabilistic predictions of a binary classifier are calibrated.
+It plots the frequency of the positive label (to be more precise, an estimation of the
+*conditional event probability* :math:`P(Y=1|predict_proba)`) on the y-axis against the
```

Comment:
The math renders as:

![Screenshot 2023-02-24 at 7 18 44 AM](https://user-images.githubusercontent.com/5402633/221177401-5b190967-8077-44dd-aacc-717da8680d9f.jpg)

We'll have to do `P(Y=1|\text{predict\_proba})` to make it render correctly.